components/calendar/calendar.js

```diff
@@ -0,0 +1,474 @@
+import '../button/button-icon.js';
+import './calendar-date.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { getDateTimeDescriptor, parseDate } from '@brightspace-ui/intl/lib/dateTime.js';
+import { getUniqueId } from '../../helpers/uniqueId.js';
+import { heading4Styles } from '../typography/styles.js';
+import { LocalizeStaticMixin } from '../../mixins/localize-static-mixin.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
```

Comment:
Is `RtlMixin` used (or will it be)?

Reply:
I think it will be - right now the keyboard arrow key navigation is backwards in rtl so I think that will have to be fixed (assuming that the expectation of rtl is pressing the left arrow key would focus on the date to the left etc.)

Reply:
I think so - our arrowkeys mixin has some similar logic for RTL.